apps/tx-builder/src/components/Builder.tsx

```diff
@@ -57,6 +52,33 @@ const getInputHelper = (input: any) => {
   }
 };
 
+// Same regex used for web3@1.3.6
+const paramTypeNumber = new RegExp(/^(u?int)([0-9]*)$/);
+
+// This function is used to apply some parsing to some value types
+const parseInputValue = (input: any, value: string): string => {
+  // If there is a match with this regular expression we get an array value like the following
+  // ex: ['uint16', 'uint', '16']. If no match, null is returned
+  const isNumberInput = input.type.match(paramTypeNumber)
```

Comment:
It's better to use `test` because it returns a boolean.
```
const isNumberInput = paramTypeNumber.test(input.type)
```

Reply:
Ok in any case I will need to extract the last numbers to get the bitWidth

Reply:
Oh, didn't notice that. I would rename the variable into `numberMatch` or something.

Reply:
I did the `test` for checking that is a number and the `match` when I extract the `bitWidth` to be more explicit. Code readability is improved